src/game_constants.h

```diff
@@ -88,6 +88,18 @@
 /** Maximum range at which ranged attacks can be executed */
 #define RANGE_HARD_CAP 60
 
+/** Player view distance FLOAT */
+constexpr float player_view_distance = 60.0f;
```

Comment:
Please hoist to a header file shared by lightmap.cpp and player.cpp, map.h would work.